ext/legacydedupefinder/legacydedupefinder.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+require_once 'legacydedupefinder.civix.php';
+
+use CRM_Legacydedupefinder_ExtensionUtil as E;
+
+/**
+ * Implements hook_civicrm_config().
+ *
+ * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_config/
+ */
+function legacydedupefinder_civicrm_config(&$config): void {
+  _legacydedupefinder_civix_civicrm_config($config);
+  \Civi::dispatcher()->addListener('hook_civicrm_findExistingDuplicates', ['\Civi\LegacyFinder\Finder', 'findExistingDuplicates'], -5);
```

Comment:
I didn''t think this was still the right way - but my reading of https://docs.civicrm.org/dev/en/latest/hooks/usage/symfony/ is that unless it's a BAO class or in a very specific location then it still is

Reply:
@mattwire I just tried & failed to switch over  - see https://github.com/civicrm/civicrm-core/pull/31959

I tried (very reluctantly) renaming the class to a `BAO` class - as that seemed slightly less inaccurate than an apiv4 event subscriber class per

![image](https://github.com/user-attachments/assets/bdd9d684-748b-450a-a7c7-646cdaa4c7bc)

But no joy.

Note `HookInterface` doesn't seem right as it has no ability to set weight